Makefile

```diff
@@ -38,7 +38,7 @@ GOROOT:=
 GO111MODULE=on
 export PATH GOROOT GO111MODULE
 # work around broken PATH export
-SHELL:=env PATH=$(PATH) $(SHELL)
+SHELL:=env PATH="$(PATH)" $(SHELL)
```

Comment:
I'm pretty sure this adds literal quotes, this is a makefile, not a shell, so it's setting a literal single argument of `PATH="`+ contents of $PATH + `"` to `env`

Reply:
Yes. Make is not as cumbersome as shell regarding quotes. But this will end up in `sh`, thus should suffice shell's syntax.

Reply:
the value `$(PATH)` winds up in sh, but if you add the quotes then the first and last path elements are actually just incorrect.

you can see how this PR breaks the path in CI for example.

Reply:
ah just saw your edit, let's try the space subsitution trick in CI, I'll take a look locally as well.